text/0000-refract-1.0.md

```diff
@@ -0,0 +1,56 @@
+- Start Date: 2015-08-29
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+
+The document provides some thoughts on how we can move forward to version 1.0 of Refract. This document is not proposing we move to 1.0 immediately, but to rather start putting steps in place to get us there quickly and being intentional about it.
+
+# Motivation
+
+As Refract is moving into production environments, we need to keep the following in mind:
+
+1. We want to have as a concise of a spec as necessary for newcomers
+1. We want to introduce breaking changes thoughtfully, which is what SemVer encourages. We currently can't do SemVer correctly without moving to 1.0.
+1. We want to start registering media types for Refract representations
+1. Moving this to production means people are actually going to start using this spec in the wild, and we need to prepare for feedback
+
+With this in mind, I'd like to start now thinking through the steps on how to prepare. This design outlines some small steps. Additionally, this document is open to additional suggestions in what we can do.
+
+# Detailed design
+
+## Versioning Namespace Documents
+
+In addition to moving to 1.0, we need to actually start versioning namespace documents. The reason for this is that we do not want a breaking change in a namespace to require a major version bump for the entire spec.
```

Comment:
@kylef You're probably right there. We can then link from the README accordingly. Sounds good to me :)